roles/openshift_node_upgrade/tasks/rpm_upgrade.yml

```diff
@@ -12,3 +12,8 @@
 - name: Ensure python-yaml present for config upgrade
   package: name=PyYAML state=present
   when: not openshift.common.is_atomic | bool
+
+- name: Install Node service file
+  template:
+    dest: "/etc/systemd/system/{{ openshift.common.service_type }}-node.service"
+    src: "{{ openshift.common.service_type }}-node.service.j2"
```

Comment:
The `openshift_node_upgrade` duplicates a lot of files from `openshift_node`. Would be more beneficial to merge both roles into so no files between them are duplicated. So `openshift_node_upgrade`'s duplicated files must be 1:1 with the ones in `openshift_node`. I did not want to introduce any symlinks in case we put the roles into galaxy. However, given the current situation I am for the symlinks before the roles are refactored.